docs/guides/continuous-integration/github-actions.mdx

```diff
@@ -349,98 +344,45 @@ on: push
 
 jobs:
   install:
-  # ...
+  # ... omitted install job
 
-  ui-chrome-tests:
+  cypress-run:
     runs-on: ubuntu-22.04
-    container: cypress/browsers:node12.18.3-chrome87-ff82
```

Comment:
If you remove the container, then users may follow this example and run into the open issue 
- https://github.com/cypress-io/cypress/issues/25040

I added the follow text to the GHA [readme](https://github.com/cypress-io/github-action#parallel)

"During staged rollout of a new GitHub-hosted runner version, GitHub may provide a mixture of current and new image versions used by the container matrix. It is recommended to use a [Docker image](#docker-image) in the parallel job run which avoids any Cypress Cloud errors due to browser major version mismatch from the two different image versions. A [Docker image](#docker-image) is not necessary if testing against the default built-in Electron browser because this browser version is fixed by the Cypress version in use and it is unaffected by any GitHub runner image rollout."

Reply:
Thanks for the heads up. In this particular example they won't run into the issue because it will use the electron browser. However, I'll expand the section below on using the same containers to include this recommendation if they are using other browsers